pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -25,48 +25,49 @@ import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 	"k8s.io/kubernetes/pkg/scheduler/internal/cache"
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 	st "k8s.io/kubernetes/pkg/scheduler/testing"
 )
 
-var (
-	hardSpread = v1.DoNotSchedule
-	softSpread = v1.ScheduleAnyway
-)
+var cmpOpts = []cmp.Option{
```

Comment:
This is neat!

Reply:
Yeah... This is as good as it can get in terms of obtaining good diffs.